spec/api/v1/schemas/profiles.rb

```diff
@@ -10,8 +10,11 @@ module Profiles
 
         PROFILE = {
           type: 'object',
-          required: %w[parent_profile_id],
+          required: %w[name parent_profile_id],
```

Comment:
name isn't actually required. It will be copied from the parent profile if it's not provided.